docs/about-us.rst

```diff
@@ -0,0 +1,61 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _about-us:
+
+********
+About Us
+********
+
+``meson-python`` is a project that extends the Meson_ build system, making it
+possible to it for Python_ packaging. It is 100% open source and is available
+under the permissive `MIT (Expat) license`_. The project was started in 2021,
+and moved under the umbrella of `The Meson Build System organization`_ in 2022.
```

Comment:
Mentioning the Meson license here is probably important as it is obviously as required dependency and its license AFAIK is not compatible with the MIT license https://mesonbuild.com/legal.html

Reply:
Good, idea, thanks! Will make the change.